### PR TITLE
Adding NeoKey Snap code.

### PR DESCRIPTION
**Pull request by kattni**

### CODE REVIEW

File: NeoKey_5x6_Ortho_Snap-Apart/2x4_NeoPixel_demo/code.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""
+MeoKey 4x2 Ortho Snap-Apart simple key press NeoPixel demo.
```
Comment (jepler) [inline NeoKey_5x6_Ortho_Snap-Apart/2x4_NeoPixel_demo/code.py:2]: "NeoKey"

File: NeoKey_5x6_Ortho_Snap-Apart/Full_sheet_NeoPixel_demo/code.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+"""
+MeoKey 5x6 Ortho Snap-Apart simple key press NeoPixel demo.
```
Comment (jepler) [inline NeoKey_5x6_Ortho_Snap-Apart/Full_sheet_NeoPixel_demo/code.py:2]: ditto

**Review by jepler — CHANGES_REQUESTED** [review]
Some typos.  Didn't test, but everything looks plausible.  Meow.

File: NeoKey_5x6_Ortho_Snap-Apart/2x4_NeoPixel_demo/code.py (hunk 1 above)
Comment (kattni) [reply NeoKey_5x6_Ortho_Snap-Apart/2x4_NeoPixel_demo/code.py]: Good catch. Thanks.

**Review by ladyada — APPROVED** [review]

**Pull request merged**